pennylane/operation.py

```diff
@@ -300,48 +394,92 @@ def hash(self):
         """int: returns an integer hash uniquely representing the operator"""
         return hash((str(self.name), tuple(self.wires.tolist()), _process_data(self)))
 
-    @classmethod
-    def _matrix(cls, *params):
-        """Matrix representation of the operator
-        in the computational basis.
+    # pylint:disable=unused-argument
+    @staticmethod
+    def compute_matrix(*params, **hyperparams):
```

Comment:
I mentioned this in #2045 just now, but I have a feeling that putting this here simply applies the pylint comment to the entire class? Would be safer to do

```python
def compute_matrix(*params, **hyperparams):
    # pylint:disable=unused-argument
```

Reply:
I tried to find all other instances too!